docs/develop/api_for_vms.rst

```diff
@@ -0,0 +1,48 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
+2. Start Date
+3. Venue
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
+
+In case of a ``GET`` request, a list containing the details of all meetups
+in the ascending order of their dates will be returned.
+
+In case of a ``POST`` request, a list containing the details of all meetups
+greater than or equal to the posted date will be returned.
+
+Details of the API
+------------------
+
+1. Use Case : Send event details
+
+2. Endpoint : Volunteers can contribute accordingly
```

Comment:
Endpoint: Is usually the link where the request is sent. `localhost/meetup/api/v1/request_meetup_data/` in this case.  
You can have the `UseCase: Send event details for volunteers to contribute accordingly.` Instead